backend/tarsy/controllers/chat_controller.py

```
@@ -0,0 +1,486 @@
+    - `available: false` with reason if unavailable
+    - `chat_id` if chat already exists
+    
+    **Availability Criteria:**
+    - Session must exist
+    - Session must be in a terminal state (COMPLETED, FAILED, or CANCELLED)
+    - Chain must have chat_enabled=true
+    """,
+)
+async def check_chat_availability(
+    session_id: str = Path(..., description="Session identifier"),
+    chat_service: Annotated[ChatService, Depends(get_chat_service)] = None,
```

Ruff (0.14.3)

Unused function argument: `chat_service`

(ARG001)